rapids/rapids.sh

```diff
@@ -1,101 +1,67 @@
 #!/bin/bash
 
+# Copyright 2019,2020,2021,2022,2024 Google LLC
```

Comment:
@jakirkham - could I get a review of this file, please?

It might be easier to look at here:
https://github.com/cjac/initialization-actions/blob/rapids-20240806/rapids/rapids.sh

I've never really used this software, so I don't know what the installer and test suite should look like.  I'd appreciate it if you could offer any insight.

Thanks for your time!

C.J.

Reply:
This was done in this review: https://github.com/GoogleCloudDataproc/initialization-actions/pull/1219#pullrequestreview-2238944546